qemu/tests/block_stream.py

```diff
@@ -11,16 +12,27 @@
 class BlockStreamTest(blk_stream.BlockStream):
 
     def get_image_size(self, image_file):
-        qemu_img = utils_misc.get_qemu_img_binary(self.params)
-        cmd = "%s info %s" % (qemu_img, image_file)
-        info = utils.system_output(cmd)
-        size = re.findall("(\d+) bytes", info)
+        try:
+            qemu_img = utils_misc.get_qemu_img_binary(self.params)
```

Comment:
get image size via monitor command is enough in this case, I think we can safely drop it.

Reply:
@xutian Thanks, but as Yanan has mentioned, it does not work for rhel6, so need to keep it anyway.